app/src/main/java/fr/free/nrw/commons/utils/PermissionUtils.java

```diff
@@ -1,121 +1,146 @@
 package fr.free.nrw.commons.utils;
 
+import android.Manifest;
 import android.app.Activity;
 import android.content.Intent;
 import android.content.pm.PackageManager;
 import android.net.Uri;
+import android.os.Build;
 import android.provider.Settings;
 
 import androidx.annotation.StringRes;
 import androidx.core.content.ContextCompat;
 
 import com.karumi.dexter.Dexter;
+import com.karumi.dexter.MultiplePermissionsReport;
 import com.karumi.dexter.PermissionToken;
-import com.karumi.dexter.listener.PermissionDeniedResponse;
-import com.karumi.dexter.listener.PermissionGrantedResponse;
 import com.karumi.dexter.listener.PermissionRequest;
-import com.karumi.dexter.listener.single.BasePermissionListener;
+import com.karumi.dexter.listener.multi.MultiplePermissionsListener;
 
 import fr.free.nrw.commons.CommonsApplication;
 import fr.free.nrw.commons.R;
+import java.util.List;
 
 
 public class PermissionUtils {
 
+    public static String[] PERMISSIONS_STORAGE = isSDKVersionScopedStorageCompatible() ?
+        isSDKVersionTiramisu() ? new String[]{Manifest.permission.READ_MEDIA_AUDIO,
```

Comment:
I removed the AUDIO and VIDEO lines here and in the manifest, it seems to work just as well. :-)
Would you mind removing these from the pull request? Or I can do it after merging this pull request, if you don't have time.

Reply:
Done, please pull the latest changes in this PR